MAINTAINERS.md

```diff
@@ -1,3 +1,19 @@
+<img src="https://opensearch.org/assets/brand/SVG/Logo/opensearch_logo_default.svg" height="64px"/>
+
+- [Maintainers](#maintainers)
+- [Zenhub Process Workflow](#zenhub-process-workflow)
+  - [What is Zenhub](#what-is-zenhubhttpswwwzenhubcom)
+  - [Dev Deployment](#dev-deployment)
+  - [Setting up Zenhub](#setting-up-zenhub)
+  - [Sprint Board aka Zenhub board aka Kanban board](#sprint-board-aka-zenhub-board-aka-kanban-board)
```

Comment:
Is `aka` formal enough? Maybe capitalize it  or change it to `/`?

Reply:
I am okay with both.

Reply:
Lets stick to a single term as this is the documentation where we declare what we do, I think `Sprint board` works well enough 
```suggestion
  - [Sprint Board](#sprint-board)
```